include/openssl/crypto.h

```diff
@@ -24,6 +24,7 @@
 # include <openssl/opensslv.h>
 # include <openssl/ossl_typ.h>
 # include <openssl/opensslconf.h>
+# include <openssl/symhacks.h>
```

Comment:
If someone include `openssl/cryptoerr.h` only, this won't help.  You should look in `crypto/cpt_err.h`...  or, for that matter, `crypto/params.c` in certain PRs :wink:

Reply:
I can think of two alternatives

- Change mkerr to put it in EVERY generated err file
- Change mkerr to have a special flag to HACK it into just that one file.

The latter is too gross to consider.

If you have a better idea that'd be great.

Editing auto-generated files is horrific.  That's the whole reason why I rewrote mkerr originally :)

Reply:
> * Change mkerr to put it in EVERY generated err file

That seems like the better alternative.

Reply:
okay, done.